commercial/app/views/hosted/guardianHostedVideo.scala.html

```diff
@@ -31,6 +31,16 @@ <h1 class="hosted__heading hosted-tone--renault">@page.video.title</h1>
                             <img class="hostedbadge__logo" src="@page.logoUrl" alt="logo Renault">
                         </div>
                     </div>
+                    <div class="hosted-next-autoplay js-hosted-next-autoplay">
+                        <div class="hosted-next-autoplay--header">
+                            <h4 class="hosted-next-autoplay--subheader">Playing next</h4>
```

Comment:
Small accessibility tip: avoid inverting subheader hierarchy. So if you can replace this H4 with a DIV, so much the better.